### ISSUE TITLE
toastPosition not working

### ISSUE THREAD

**Issue by unleed-l** [koukibadr/Cherry-Toast#62]
toastPosition property is not working as expected, toast still appearing at the top (tested on macOS).

```
void warningToast({
    required String text,
    String? actionText,
    Function? actionHandler,
    required BuildContext context,
  }) {
    CherryToast.warning(
      title: Text(
        text,
        style: TextStyle(
          color: Colors.amber.shade700,
          fontWeight: FontWeight.w600,
        ),
      ),
      toastDuration: duration,
      shadowColor: Colors.transparent,
      toastPosition: Position.bottom,
      displayCloseButton: false,
      animationType: AnimationType.fromTop,
      action: actionText != null
          ? Text(
              actionText,
              style: TextStyle(
                color: Colors.blue,
              ),
            )
          : null,
      actionHandler: actionHandler,
    ).show(context);
  }
```

**Comment by elssiany** [comment]
Hello, I have created this [PR](https://github.com/koukibadr/Cherry-Toast/pull/65) adding the toast positions, you can see the solution, regards.

**Comment by unleed-l** [comment]
> Hello, I have created this [PR](https://github.com/koukibadr/Cherry-Toast/pull/65) adding the toast positions, you can see the solution, regards.

Thanks!

**Comment by koukibadr** [comment]
A new version has been published with the new changes check it out 1.7.0 
thanks @elssiany for the new updates you bring to the package

@unleed-l feel free to close this issue
